array-methods/map-and-filter/src/__tests__/1_map_filter_exercises.test.js

```diff
@@ -28,25 +27,17 @@ tool to understand how the real prototype method works.
 🛠️ An array and a 'callback' function.
 🛠️ vanillaMap should return a new array populated with the results of calling the 
 🛠️ provided callback on every element of the input array.
-`
+`;
 
 const vanillaMap = (numArray, callback) => {
-  // Your solution here
+  return numArray.map(callback);
```

Comment:
```suggestion
  const newArray = [];
  for (const num of numArray) {
    newArray.push(callback(num));
  }
  return newArray;
}
```
Rather than using the vanilla `.map`, they were looking for a custom implementation of one like in the `forEach` lesson.

Reply:
Their solutions actually used `.forEach`—guess that's fair game now since you already wrote one of those. Makes sense to me!